shiny/bookmark/_utils.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+import orjson
+
+
+def is_hosted() -> bool:
+    # Can't look at SHINY_PORT, as we already set it in shiny/_main.py's `run_app()`
+
+    # TODO: Barret: Q: How to support shinyapps.io? Or use `SHINY_PORT` how R-shiny did
```

Comment:
Make a PR to change the name `SHINY_PORT` to something else, to avoid this naming conflict.